etc/helm/pachyderm/templates/proxy/deployment.yaml

```diff
@@ -102,8 +102,17 @@ spec:
           volumeMounts:
             - name: envoy-config
               mountPath: /etc/envoy
+            {{- if .Values.proxy.tls.enabled }}
+            - name: tls
+              mountPath: /tls
+            {{- end }}
       volumes:
         - name: envoy-config
           configMap:
             name: pachyderm-proxy-config
+        {{- if .Values.proxy.tls.enabled }}
+        - name: tls
+          secret:
+            secretName: {{ .Values.proxy.tls.secretName }}
```

Comment:
should this hook up to the helm generated `pachyderm-proxy-tls`?

Reply:
so outcome here is to keep things as they are and ask customers to create their own secret? That makes sense to me

Reply:
I think you kind of talked me into doing it like the rest of the chart.  You will have to set the name, but I think that's OK.

Reply:
I made the generated secret match the secretName, erroring when it's not set.  So there is now no magic string when generating a secret from key/cert.  Also added an error for the case where you set one of key/cert without the other (easy to spell cert as `cret` or something).